spec/helpers/form_builder_helper_spec.rb

```diff
@@ -13,10 +13,11 @@
   end
 
   describe '#continue_button' do
+    # rubocop:disable Layout/LineLength
     context 'when there is no secondary action' do
```

Comment:
Might be me reading this wrong but the wording on this test is confusing to me in this context. If there is no secondry action on the form then surely that form doesn't need its submit to go through the hoisting to the top.

I think the test is logically correct just the description of throws me a bit 🤔

Reply:
These tests are about the form builder. Secondary in this scenario is the "draft" button. We can have the green button, and most of the time next to it we will also have the grey save draft button. In some very specific steps we don't have this secondary (draft) button, like the first few steps when starting an application, and same blue pages.

<img width="511" alt="Screenshot 2023-07-13 at 11 26 54" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/687910/5a9d7120-f7ac-44cb-8ccf-6d418b2096a4">

Reply:
Ah right! I get it - was thinking about the word secondry more generally. Thanks for clearing this up for me 👍

Reply:
Yep, grey buttons are all considered secondary in terms of the design system. The form builder uses `secondary: true` in the helper, and then the applied css class is `govuk-button--secondary` thus the naming 😄